<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="titulo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>421</width>
      <height>121</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/imagenes/logo1peq.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="indent">
     <number>1</number>
    </property>
   </widget>
   <widget class="QTextBrowser" name="gui_listaPersonas">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>390</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="FramePrincipal">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>471</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="QLabel" name="gui_labelp">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>440</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Versión 1.0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_enviarWhats">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>390</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 10pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>Enviar Whatsapp</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_enviarMail">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>390</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 10pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>Enviar eMail</string>
    </property>
   </widget>
   <zorder>FramePrincipal</zorder>
   <zorder>titulo</zorder>
   <zorder>gui_listaPersonas</zorder>
   <zorder>gui_labelp</zorder>
   <zorder>btn_enviarWhats</zorder>
   <zorder>btn_enviarMail</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>471</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPrincipal">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="act_borrarBD"/>
    <addaction name="separator"/>
    <addaction name="act_datosAdmin"/>
    <addaction name="separator"/>
    <addaction name="act_salir"/>
   </widget>
   <widget class="QMenu" name="menuOpciones">
    <property name="title">
     <string>Opciones</string>
    </property>
    <addaction name="act_ajustesOri"/>
    <addaction name="act_ajustesSal"/>
    <addaction name="separator"/>
    <addaction name="act_mostrarLog"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="act_acercaDe"/>
   </widget>
   <addaction name="menuPrincipal"/>
   <addaction name="menuOpciones"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="act_borrarBD">
   <property name="text">
    <string>Ya se me ocurrira algo</string>
   </property>
   <property name="toolTip">
    <string>Ya se me ocurrira algo</string>
   </property>
  </action>
  <action name="act_salir">
   <property name="text">
    <string>Salir de la aplicación</string>
   </property>
   <property name="toolTip">
    <string>Salir de la aplicación</string>
   </property>
  </action>
  <action name="act_ajustesOri">
   <property name="text">
    <string>Ajustes origen ficheros</string>
   </property>
  </action>
  <action name="act_ajustesSal">
   <property name="text">
    <string>Ajustes salida ficheros</string>
   </property>
  </action>
  <action name="act_mostrarLog">
   <property name="text">
    <string>Registro de la última copia</string>
   </property>
  </action>
  <action name="act_datosAdmin">
   <property name="text">
    <string>Datos admin</string>
   </property>
  </action>
  <action name="act_acercaDe">
   <property name="text">
    <string>Acerca de</string>
   </property>
  </action>
  <action name="act_cumpleaneros">
   <property name="text">
    <string>Listado de cumpleañeros</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../Curso Python/Bd2Xls/gui/imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
